lib/filesystem.js

```diff
@@ -138,7 +138,7 @@ exports.readChunk = (fd, size, position, callback, retries) => {
     if (error) {
       const platform = os.platform();
 
-      if (error.code === 'ENXIO' && platform === 'darwin') {
+      if (platform === 'darwin' && (error.code === 'ENXIO' || error.code === 'EBUSY')) {
```

Comment:
For consistency, could you swap around the other error-handling code to check the `platform` before checking the `error.code` ?